src/Raven.Studio/typescript/viewmodels/resources/widgets/clusterOverviewWidget.ts

```diff
@@ -0,0 +1,288 @@
+import websocketBasedWidget = require("viewmodels/resources/widgets/websocketBasedWidget");
+import clusterDashboard = require("viewmodels/resources/clusterDashboard");
+import clusterDashboardWebSocketClient = require("common/clusterDashboardWebSocketClient");
+import virtualGridController = require("widgets/virtualGrid/virtualGridController");
+import columnPreviewPlugin = require("widgets/virtualGrid/columnPreviewPlugin");
+import virtualColumn = require("widgets/virtualGrid/columns/virtualColumn");
+import textColumn = require("widgets/virtualGrid/columns/textColumn");
+import iconsPlusTextColumn = require("widgets/virtualGrid/columns/iconsPlusTextColumn");
+import nodeTagColumn = require("widgets/virtualGrid/columns/nodeTagColumn");
+import actionColumn = require("widgets/virtualGrid/columns/actionColumn");
+import clusterTopologyManager = require("common/shell/clusterTopologyManager");
+import generalUtils = require("common/generalUtils");
+import appUrl = require("common/appUrl");
+import router = require("plugins/router");
+import moment = require("moment");
+
+class nodeStatsItem {
+    
+    nodeTag: string;
+    nodeUrl = ko.observable<string>();
+    
+    nodeType = ko.observable<clusterNodeType>();
+    nodeState = ko.observable<Raven.Client.ServerWide.RachisState>();
+
+    startTime = ko.observable<string>();
+    formattedUpTime = ko.observable<string>();
```

Comment:
sounds like computed?

Reply:
done

Reply:
This is computed. :) we have silimuar concept in operations where we compute duration. You can use that then this will be autoupdated